conda-installation.md

```diff
@@ -35,10 +40,16 @@ your jobs. The other option is to install Miniconda inside each job.
 The first option is more efficient, especially for complex installations, 
 but there may be rare situations where installing with each job is better. 
 We recommend trying the pre-installation option first.
+If it doesn't work, discuss the second option with an RCF.
 
 - Recommended: [Option 1: Pre-Install Miniconda and Transfer to Jobs](#option-1-pre-install-miniconda-and-transfer-to-jobs)
 - Alternative: [Option 2: Install Miniconda Inside Each Job](#option-2-install-miniconda-inside-each-job)
 
+This guide also discusses how to 
+["pin" your conda environment](#pinning-dependencies)
+to create a consistent, reproducible environment with specified versions of 
```

Comment:
```suggestion
to create a more consistent and reproducible environment with specified versions of 
```
Is there a way to soften the claim of reproducibility?  Pinning packages helps a lot, but there are still other things that can go wrong.